google-cloud-datastore/src/main/java/com/google/cloud/datastore/Batch.java

```diff
@@ -32,7 +33,15 @@
  * batch.add(entity2, entity3);
  * batch.submit();
  * }</pre>
+ *
+ * <p><b> WARNING: THIS CLASS MAINTAINS AN INTERNAL STATE IN TERMS OF {@link
+ * java.util.LinkedHashMap} AND {@link java.util.LinkedHashSet} WHICH GETS UPDATED ON EVERY METHOD
+ * CALL PERFORMING CRUD OPERATIONS TO RECORD THE MUTATIONS, SINCE {@link java.util.LinkedHashMap} IS
```

Comment:
two nits -
1. I would use lowercasing here (maybe just keep "WARNING" in all caps?)
2. Run on sentence, I would rephrase: (`...to record the mutations. Since LinkedHashMap...`) (swap out the comma with a period)

Reply:
fixed in fb6c899.